R/load.r

```diff
@@ -92,10 +94,10 @@
 #' load_all("./", export_all = FALSE)
 #' }
 #' @export
-load_all <- function(path = ".", reset = TRUE, recompile = FALSE,
+load_all <- function(path = ".", reset = TRUE, compile = pkgbuild::needs_compile(),
```

Comment:
I guess we need to pass the path argument to `needs_compile()` as well, sorry my previous suggestion omitted it.

```suggestion
load_all <- function(path = ".", reset = TRUE, compile = pkgbuild::needs_compile(path),
```